cmd/skopeo/utils.go

```diff
@@ -228,3 +231,15 @@ func parseImageSource(ctx context.Context, opts *imageOptions, name string) (typ
 	}
 	return ref.NewImageSource(ctx, sys)
 }
+
+func yamlUnmarshal(yamlFile string, cfg interface{}) error {
+	source, err := ioutil.ReadFile(yamlFile)
+	if err != nil {
+		return err
+	}
+	err = yaml.Unmarshal(source, cfg)
+	if err != nil {
+		return errors.WithMessage(err, fmt.Sprintf("cannot unmarshal %s", yamlFile))
```

Comment:
The rest of Skopeo usually uses `error.Wrapf(...)`. Can we stick to that? In this specific case, `errors.WithMessagef(...)` would avoid the call to `Sprintf(...)`.

Same applies to other uses of `WithMessage(...)`.

Reply:
Still applies...

Reply:
Repetita iuvant :-)